.github/workflows/build.yml

```diff
@@ -69,6 +69,31 @@ jobs:
               echo "~~~~ maturin building"
               . .env/bin/activate && maturin build
             fi
+      - name: aarch64-unknown-linux-gnu
+        uses: addnab/docker-run-action@v3
+        if: ${{ inputs.target == 'aarch64-unknown-linux-gnu' }}
+        with:
+          image: ${{ inputs.docker }}
+          options: '-e SHOULD_PUBLISH=${{ inputs.publish }} --user 0:0 -v ${{ github.workspace }}/.cargo-cache/git/db:/usr/local/cargo/git/db -v ${{ github.workspace }}/.cargo/registry/cache:/usr/local/cargo/registry/cache -v ${{ github.workspace }}/.cargo/registry/index:/usr/local/cargo/registry/index -v ${{ github.workspace }}:/build -w /build'
+          run: |
+            pwd
+            ls -lah
```

Comment:
Is this just helpful build info?

Reply:
Aye, debugging that I think originally came from the nodejs napi-rs folks, but it's been helpful enough I've kept it around.